example_sources.json

```diff
@@ -0,0 +1,5 @@
+[{
+    "version": "debug",
+    "url": "https://raw.githubusercontent.com/6r1d/iroha/iroha2_doctest_update_attempt_2/client/examples/tutorial.rs",
```

Comment:
this url is for your fork, is that right?

Reply:
Correct, for now this is a draft and in the end I'll add real examples containing this syntax.